src/View/Calendar.php

```diff
@@ -213,7 +211,7 @@ protected static function renderHeadlineDays( $days_display ): string {
 		$divider = "</th><th class='cal sortless'>";
 		$dayStr  = implode( $divider, $days_display );
 
-		return '</tr><tr>' .
+		return '<tr>' .
 		       '<th>' . __( 'Item', 'commonsbooking' ) . '</th>' .
 		       '<th>' . __( 'Location', 'commonsbooking' ) . "<th class='cal sortless'>" . $dayStr . '</th>' .
 		       '</tr></thead><tbody>';
```

Comment:
@chriwen wenn du das eh schon änderst, was hälst du davon an der Stelle die `<tr>`-Tags und schließenden `<thead>` sowie öffnenden `<tbody>` aus dieser Methode heraus zu ziehen und auf die Ebene von Zeile 93 zu schreiben. Dann finde ich das wesentlich lesbarer und konsistent mit dem in Zeile 93 bisher geschriebenen Code.

Dan ist nur der für die Headline days relevante Teil in der Methode und die allgemein Struktur bleibt beim Aufrufer klar.

Reply:
@chriwen da es so überspezfisch formuliert  habe ich es einfach mal selbst umgesetzt. Hoffe das passt für dich!